packages/hawtio/src/plugins/shared/attributes/AttributeModal.tsx

```diff
@@ -75,18 +75,20 @@ export const AttributeModal: React.FunctionComponent<{
 
   const modalTitle = `Attribute: ${attributeName}`
 
-  const modalActions = [
-    <Button key='close' variant='primary' onClick={onClose}>
-      Close
-    </Button>,
-  ]
+  const modalActions = []
+
   if (isWritable) {
     modalActions.push(
       <Button key='update' variant='danger' onClick={updateAttribute}>
-        Update
+        Save
       </Button>,
     )
   }
+  modalActions.push(
+    <Button key='close' variant='link' onClick={onClose}>
+      Cancel
```

Comment:
As I commented, `Cancel` doesn't make much sense to me. Anyway, isn't this part changed altogether with a following pull req?  Why do we bother changing this part then?